initializers/ip_addresses.yml

```diff
@@ -0,0 +1,13 @@
+#- address: 10.1.1.1/24
+#  device: server01
+#  interface: Loopback0
```

Comment:
This interface doesn't exist. I believe it should be the following:

```suggestion
#  interface: to-server02
```

Reply:
Fixed.